src/doc/trpl/traits.md

```diff
@@ -338,7 +338,8 @@ plain type parameter (like `T`).
 
 ## Default methods
 
-There’s one last feature of traits we should cover: default methods. It’s
+There’s one important feature of traits distinguishing them from "interfaces",
+familiar to many from languages like Java, C# or Go: *default methods*. It’s
```

Comment:
I felt that some comparison to interfaces would be great as an aside, because the confusion about terminology is pretty commonplace, and default methods is one of the features that clearly distinguishes traits from interfaces. Maybe we could do that without singling out any specific languages?